test/evergreen.yml

```diff
@@ -344,7 +344,13 @@ functions:
 
             # Preload tcmalloc in the Evergreen environment: this will affect
             # ALL binaries.
-            export LD_PRELOAD=$PWD/TCMALLOC_LIB/libtcmalloc.so
+            # FIXME-WT-14389 v5 toolchain clang Cmake configuration fails with tcmalloc preloading
```

Comment:
We'll need to coordinate the merge of this ticket with WT-14389, otherwise we'll get 'outdates fixme' errors.

Reply:
I agree, I will wait for WT-14389 to get merged first.

Reply:
Since we do need to perform additional perf testing for WT-14389 I suggest merging this one first